src/constant/study-room.ts

```diff
@@ -1,7 +1,15 @@
 export const STUDY_ROOM_STEP = {
+  WAITING: '대기',
   PLANNING: '계획',
   STUDYING: '스터디',
   RETROSPECT: '회고',
   RESTING: '휴식',
   COMPLETED: '완료',
 };
+
+export const STUDY_ROOM_STEP_TO_TIMESET = {
```

Comment:
timeSet을 다른 파일에서 두 단어로 표현하고 있어서 여기서도 이렇게 표편해주는게 일관적일 것 같아요~
```suggestion
export const STUDY_ROOM_STEP_TO_TIME_SET = {
```